src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java

```diff
@@ -23,16 +23,11 @@
 
 	private static final Logger LOGGER = LoggingUtils.getLogger();
 
-	/**
-	 * Constructor.
-	 */
-	public MissingImageDeliveriesController() {
-	}
-
 	@PostMapping("${uk.gov.companieshouse.chd.order.api.mid}")
 	public ResponseEntity<MissingImageDeliveriesDTO> createMissingImageDelivery(final @Valid @RequestBody MissingImageDeliveriesDTO midDTO,
 			HttpServletRequest request) {
-		Map<String, Object> logMap = LoggingUtils.createLoggingDataMap(COMPANY_NUMBER_LOG_KEY, "");
+		Map<String, Object> logMap = LoggingUtils.createLoggingDataMap(COMPANY_NUMBER_LOG_KEY,
+				midDTO.getCompanyNumber());
```

Comment:
This shouldn't have tabs in it, it should use spaces instead. If the entire class is like this, we should raise the formatting as a separate Jira issue. If not, this should be re-formatted.

Reply:
Arghh! Caught out by this again! I'll see whether this affects whole files or just the bit I have changed. 👍

Reply:
I'm afraid this issue seems to affect entire files. I have raised a JIRA https://companieshouse.atlassian.net/browse/GCI-1617 to deal with the formatting in a separate PR.